torchvision/datasets/camvid.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import print_function
+
+import os
+import torch
+import torch.utils.data as data
+import numpy as np
+from PIL import Image
+from torchvision.datasets.folder import is_image_file, default_loader
+
+
+classes = ['Sky', 'Building', 'Column-Pole', 'Road',
+           'Sidewalk', 'Tree', 'Sign-Symbol', 'Fence', 'Car', 'Pedestrain',
+           'Bicyclist', 'Void']
+
+# weights when using median frequency balancing used in SegNet paper
+# https://arxiv.org/pdf/1511.00561.pdf
+# The numbers were generated by https://github.com/yandex/segnet-torch/blob/master/datasets/camvid-gen.lua
+class_weight = [0.58872014284134, 0.51052379608154, 2.6966278553009, 0.45021694898605, 1.1785038709641,
+                0.77028578519821, 2.4782588481903, 2.5273461341858, 1.0122526884079, 3.2375309467316,
+                4.1312313079834, 0]
+# mean and std
+mean = [0.41189489566336, 0.4251328133025, 0.4326707089857]
+std = [0.27413549931506, 0.28506257482912, 0.28284674400252]
+
+class_color = [
+    (128, 128, 128),
+    (128, 0, 0),
+    (192, 192, 128),
+    (128, 64, 128),
+    (0, 0, 192),
+    (128, 128, 0),
+    (192, 128, 128),
+    (64, 64, 128),
+    (64, 0, 128),
+    (64, 64, 0),
+    (0, 128, 192),
+    (0, 0, 0),
+]
+
+
+def make_dataset(dir):
+    images = []
+    for root, _, fnames in sorted(os.walk(dir)):
+        for fname in fnames:
+            if is_image_file(fname):
+                path = os.path.join(root, fname)
+                item = path
+                images.append(item)
+    return images
+
+
+def LabelToLongTensor(pic):
```

Comment:
could you use the lower-case convention for functions, and maybe make these 2 functions private (starting with an `_`)?